documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -51,8 +51,15 @@ npm install dotenv
 ...and invoke it before running the built app:
 
 ```diff
--node build
-+node -r dotenv/config build
+- node build
++ node -r dotenv/config build
```

Comment:
is this change needed? the text above says to invoke dotenv before running the *built*  app, but this is during build.
Sveltekit uses vite which has dotenv builtin https://vitejs.dev/guide/env-and-mode

Reply:
The default build directory is `./build`.

`node build` is how you start the `adapter-node` application.

https://kit.svelte.dev/docs/adapter-node#deploying

```
📦build
 ┣ 📂client
 ┣ 📂prerendered
 ┣ 📂server
 ┣ 📜env.js
 ┣ 📜handler.js
 ┣ 📜index.js
 ┗ 📜shims.js
```

Reply:
oh right :see_no_evil: